dev/com.ibm.ws.concurrent.mp_fat/test-applications/MPConcurrentCDIApp/src/concurrent/mp/fat/cdi/web/MPConcurrentCDITestServlet.java

```diff
@@ -209,21 +212,30 @@ public void testCDIContextPropagationAcrossMultipleStagesBeanAccessedFirst() thr
     }
 
     /**
-     * Propagate CDI context across completion stages. Access (and modify) a request scoped bean
-     * within the stages, but not before.
+     * Propagate CDI context to a completion stage. Access (and modify) a request scoped bean
+     * within the stage, but not before the context snapshot is taken. Lazily enlisting CDI beans
+     * should be rejected.
```

Comment:
Change
'Propagate CDI context to a completion stage. Access (and modify) a request scoped bean  within the stage, but not before the context snapshot is taken. Lazily enlisting CDI beans should be rejected.'
to
'Propagate CDI context to a completion stage. Access (and modify) a request scoped bean within the stage but not before the context snapshot is taken. Lazily enlisting CDI beans must be rejected.'